configs/lip-a2/microscope.j5

```diff
@@ -41,83 +85,114 @@
                 //More proper homed scheme
-                "G10 L2 P1 X-297.000 Y-197.000 Z-3.000",
+                //"G10 L2 P1 X-297.000 Y-197.000 Z-3.000",
+                //8/5 adjustment
+                //"G10 L2 P1 X-475.200 Y-315.200 Z-4.800",
+                //gearbox + 0.9 motor
+                "G10 L2 P1 X-475.200 Y-315.200 Z-48.000",
                 "G10 L2 P2 X0 Y0 Z0",
 
-                //Homing adjustments to account for gearbox
-                "$26=250", "$27=15.000",
-            ],
-            "rc_post_home": [
                 //Select WCS2
                 "G55",
             ]
         },
         "scalars": {
-            "x": 1.0,
-            "y": 1.0,
-            "z": 5.0,
+            "x": 1.6,
+            "y": 1.6,
+            "z": 1.6,
         },
         /*
         Actual limits should be:
         X: 0.0 to 300.0, homed at -297 (12)
         Y: 0.0 to 200.0, homed at -197 (12)
         Z: -60.0 to 0.0, homed at -2.4
         */
-        /*
-        TODO: need a better way to recover in GUI
-        Disable until then
-        "#soft_limits": {
-            "xmin": 2.5,
-            "xmax": 297.5,
-            "ymin": 2.5,
-            "ymax": 197.5,
-            "zmin": -58.5,
-            "ymax": -2.5,
-        },
-        */
-        //Token limits for now
-        "!soft_limits": {
-            "xmin": 0.0,
-            "xmax": 300.0,
-            "ymin": 0.0,
-            "ymax": 200.0,
-            "zmin": -60.0,
-            "ymax": -0.0,
-        },
-    },
-    "imager": {
-        "source":"gst-toupcamsrc",
-        "width": 5440,
-        "height": 3648,
-        "source_properties": {
-            //0 => mode 5440x3648
-            "esize": 0
+        //"home_soft_limits": true,
+        "soft_limits": {
+            //machine: 0.0 to 300.0
+            "xmin": 10.0,
+            "xmax": 290.0,
+            //machine: 0.0 to 200.0
+            "ymin": 10.0,
+            "ymax": 190.0,
+            //machine: -67.5 to +3.0
+            //LIA-TT-1 is well within this range
+            "zmin": -65.0,
+            "zmax": +1.0,
         },
-        "source_properties_mod": {
-            //In us. Can go up to 15 sec which is impractical for typical usage
-            "expotime": {
-                "max": 200000
-            },
-        },
-        "scalar": 0.5,
-        //Uncomment to save snapshots and scans 8-bit lossless
-        //Note: raw and 12 bit support is much more involved / experimental
-        //"save_extension": ".tif",
-        //5440 - 1396 * 2 = 2648
-        "crop_pixels": {
-            "top": 500,
-            "bottom": 900,
-            "left": 1596,
-            "right": 1396,
-        },
-        "um_per_pixel_raw_1x": 1.321752266,
     },
+
+    //E3ISPM20000KPA
+    "imager": {
+        "source":"gst-toupcamsrc",
+        "width": 5440,
+        "height": 3648,
+        "source_properties": {
+            "esize": 0
+        },
+        "scalar": 0.5,
```

Comment:
We should review how much usable resolution / sensor size. But I imagine this is a fine starting point

Reply:
👍🏼